packages/keymaster/src/db-wallet-json.js

```diff
@@ -1,8 +1,32 @@
 import fs from 'fs';
+import crypto from 'crypto';
```

Comment:
I suggest copying this to a new file db-wallet-json-enc.js, and reverting db-wallet-json.js